Fixes (#6734) CPU Reserved Should Be 1024 Per VCPU ECS Container Insi… (#7777)
open-telemetry/opentelemetry-collector-contrib: CHANGELOG.md

 - `prometheusreceiver`: Correctly map description and units when converting
  Prometheus metadata directly to pdata. (#7748)
 - `sumologicexporter`: fix exporter panics on malformed histogram (#7548)

> - `awsecscontainermetrics`: Fixes (#6734) CPU Reserved Should Be 1024 Per VCPU ECS Container Insights

Reviewer: ```suggestion
- `awsecscontainermetrics`: CPU Reserved is now 1024/vCPU for ECS Container Insights (#6734) 
```

Author: changed
